Makefile

```diff
@@ -1,9 +1,4 @@
-TEST_DB_HOST:=
-TEST_DB_PORT:=
-TEST_DB_USER:=
-TEST_DB_PASSWORD:=
-TEST_DB_NAME:=
-
+.PHONY: test
```

Comment:
Would it make sense to put every target in .PHONY? I'm finding Makefiles with Go a little confusing because Go already has it's own command line tools.

Reply:
i think its only needed when a directory exists with the same name as a target, but i'm 100% comfortable with it either

Reply:
I think we can leave it as is since it's working